vector/src/main/res/values-fr/strings.xml

```diff
@@ -290,7 +290,7 @@
     <string name="low_priority_header">Priorité basse</string>
     <string name="direct_chats_header">Discussions</string>
     <string name="template_local_address_book_header">Contacts ${app_name}</string>
-    <string name="matrix_only_filter">Contacts Matrix uniquement</string>
+    <string name="template_matrix_only_filter">Contacts ${app_name} uniquement</string>
```

Comment:
Can you please revert this change? the request #248 has been cancelled for the moment